packages/frontend/src/components/message/MessageList.tsx

```diff
@@ -308,16 +308,29 @@ export default function MessageList({ accountId, chat, refComposer }: Props) {
       log.debug('scrollTo: scrollToMessage: ' + scrollTo.msgId)
 
       const domElement = document.getElementById(scrollTo.msgId.toString())
```

Comment:
maybe adding a prefix to the id attribute so it is not only a number might be a good idea?
Or we change it to a custom attribute like `x-msg-id` and look for it via `document.querySelector`, though I'm not sure what is faster.

We could also call `.querySelector` on the DOM element of the message list, so that it only searches relevant nodes and that makes `.querySelector` probably faster.
But I'm not sure if the speed difference in this case is even a big concern.